scripts/seed.js

```diff
@@ -120,12 +190,10 @@ async function createCircles() {
 
         await body.update({ shadow_circle_id: membersCircle.id });
 
-        circles.board.push(boardCircle);
         circles.members.push(membersCircle);
 
         if (body.type === 'antenna') {
             circles.antennaMembersCircle = membersCircle;
-            circles.antennaBoardCircle = boardCircle;
         }
```

Comment:
`antennaMembersCircle` is unused. You can remove the whole if block.

Reply:
Indeed, I never looked at that. Are you not using it in your updates to the seeder?

Reply:
nope, I needed to deal with the other one, but you automated the creation of it (and generalized it too). Happy to see that block gone;